scripts/validate_docstrings.py

```diff
@@ -497,6 +498,20 @@ def parameter_desc(self, param):
                 desc = desc[: desc.index(full_directive)]
         return desc
 
+    def has_proper_punctuation(self, param):
+        """Return True if a parameter description is terminated correctly.
```

Comment:
I think this first paragraph is unnecessary. Also, we start docstrings in the next line after the quotes.